hypha/static_src/src/javascript/apply/application-form.js

```diff
@@ -2,6 +2,15 @@
 
     'use strict';
 
+    let symbol = $(document.currentScript).attr('symbol');
+    const amount = $('#requested-amount');
+    $(amount[0]).css({paddingLeft: '20px'});
```

Comment:
This is raise an error if `amount` is null. Also this is executed on every page request, not ideal.